src/stc/ScintillaWX.cpp

```diff
@@ -844,6 +845,31 @@ sptr_t ScintillaWX::WndProc(unsigned int iMessage, uptr_t wParam, sptr_t lParam)
       case SCI_GETDIRECTPOINTER:
             return reinterpret_cast<sptr_t>(this);
 
+#ifdef __WXMSW__
+      // ScintillaWin
+      case WM_IME_STARTCOMPOSITION: 	// dbcs
```

Comment:
```suggestion
      case WM_IME_STARTCOMPOSITION:
```
Trivial, but these "dbcs" comments just don't seem helpful to me.

Reply:
This, and the `if (false)` above, are just trying to make minimal changes to original SciWin code. Should I remove them for better readability or comment it out for minimal change?